test/scalability/results/.gitignore

```diff
@@ -1 +1,2 @@
 *.csv
```

Comment:
I think I'd prefer having all the git ignores in the root/.gitignore, using the full path instead, so `/test/scalability/results/*.png`. Looking at the exiting large number of .gitignore files this might be a lost battle though.

Reply:
✅